bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/dto/WebhookEvent.java

```diff
@@ -13,8 +13,8 @@
 package org.openhab.binding.netatmo.internal.api.dto;
 
 import java.time.ZonedDateTime;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.LinkedHashSet;
```

Comment:
The method returned duplicated entries (an object id was returned twice which is useless in that case and caused negative effects). 

This is fixed with using a LinkedHashSet instead of an ArrayList. The LinkedHashSet keeps the order like a List but guarantees that there are no duplicated entries.